dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/handle/AbstractHandle.java

```diff
@@ -22,5 +22,8 @@
 public abstract class AbstractHandle implements Handle {
 
   @Override
-  public void close() throws IOException {}
+  public void close() throws IOException {
+    // Intentionally empty to simplify implementation of some Handles.
+    // If your subclass needs to release resources, consider implementing Handle directly.
```

Comment:
I'm not sure we need any comment here. The `close()` comes from `Closeable` and it's already well documented. 

```
Closes this stream and releases any system resources associated with it. If the stream is already closed then invoking this method has no effect.
As noted in AutoCloseable.close(), cases where the close may fail require careful attention. It is strongly advised to relinquish the underlying resources and to internally mark the Closeable as closed, prior to throwing the IOException.
```
As an option we can use `{@inheritDoc}`,